CourtFinder/endpoints/users/routes.py

```diff
@@ -80,3 +80,84 @@ def register():
             return render_template('users/profile.html', user=new_user)
 
     return render_template('users/register.html', form=form)
+<<<<<<< HEAD
+
+
+@users.route('/UpdateProfile/<id>', methods=['GET','POST'])
+def updateProfile(id):
+    if request.method == 'POST':
+        user = User.query.filter_by(id=id).first()
+        user.first_name = request.form.get('inputFirstName')
+        user.last_name = request.form.get('inputLastName')
+        user.favorite_court = request.form.get('inputFavoriteCourt')
+        user.height = request.form.get('inputHeight')
+        user.weight = request.form.get('inputWeight')
+        user.skill_level = request.form.get('inputSkillLevel')
+        db.session.commit()
+        if request.form.get('inputPassword') == None:
+            flash('Password wasnt changed', 'danger')
+        if request.form.get('inputPassword') == request.form.get('inputConfirmPassword'):
+            user.password = request.form.get('inputPassword')
```

Comment:
you need to encrypt password before you insert it into db. If you dont, nothing will be inserted, passwd will be blank and you might not be able to login.

Reply:
take a look at line 60/`hashed_pass` var assignment, in the /register route.

Reply:
Also theres an issue here. Line 99 will execute if a user doesnt enter a password, because `None==None` and a blank password will inserted into the db and lock you out.

Easy solution is change this ton an else